holoviews/plotting/mpl/element.py

```diff
@@ -711,40 +749,3 @@ def update_frame(self, key, ranges=None, element=None):
             self._adjust_legend(element, axis)
 
         self._finalize_axis(key, ranges=ranges)
-
-
-
-class DrawPlot(ElementPlot):
```

Comment:
Good to see a class go. Think we have any backward compatibility concerns about this though?